code/src/index.css

```diff
@@ -1,13 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,400;1,600&family=Source+Sans+Pro&display=swap');
+
+/* --------- GENERAL ------------------------------------ */
+
+:root {
+  --lightlavender: rgb(183, 139, 255);
+  --darklavender: rgb(124, 72, 208);
+  --mint: rgb(135, 255, 205);
+  --lightgray: #fafafa;
+  --darkgray: rgb(66, 66, 66);
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-size: 15px;
+  font-family: 'Source Sans Pro', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin: 0 auto;
+  background: #f6f6f6;
+  overflow: hidden;
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.layout {
+  display: grid;
+  gap: 30px;
+  justify-items: center;
+  height: 100vh;
+}
+
+/* --------- HEADER ------------------------------------ */
+
+.layout-header {
+  font-size: 2.7rem;
+  align-self: end;
+  font-family: 'JetBrains Mono', monospace;
+  font-style: italic;
+  font-weight: 600;
+  background-image: linear-gradient(175deg,var(--lightlavender),var(--mint));
+  background-repeat: no-repeat;
+  background-position: 0% 100%;
+  background-size: 100% 4px;
+  padding-bottom: 5px;
+  letter-spacing: 3px;
+}
+
+/* --------- SURVEY ------------------------------------ */
+
+.layout-content {
+  align-self: start;
+  width: 92%;
+  max-width: 500px;
+}
+
+.survey {
+  padding: 20px;
+  justify-content: center;
+  border: 2px black dashed; 
+  background: var(--lightgray);
+  min-height: 265px;
+}
+
+input, progress {
+  accent-color: var(--lightlavender);
+}
+
+.counter-text {
+  color: var(--darkgray);
+  margin-bottom: 0px;
+  margin-top: 0px;
+  font-size: 0.9rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* ------ Buttons ------ */
+
+.button {
+  background-color: var(--mint);
+  color: black;
+  border: 1px black;
+  padding: 4px 10px;
+  font-family: inherit;
+  font-size: inherit;
+  width: 100%;
+  margin-top: 2em;
+  margin-bottom: 2em;
+  }
+
+.button:focus {
+  }
+
+.button:hover {
+  }
+
+.button:active {
+  }
```

Comment:
I guess you don't need this because they are empty.

Reply:
I was thinking to style these states of the buttons too so I left them there. Perhaps I should comment them out for now 🤔